addons/storyshots/storyshots-core/package.json

```diff
@@ -26,6 +26,9 @@
   },
   "dependencies": {
     "@storybook/addons": "5.2.0-beta.22",
+    "@types/glob": "^7.1.1",
+    "@types/jest": "^24.0.16",
+    "@types/jest-specific-snapshot": "^0.5.3",
```

Comment:
@emilio-martinez @kroeder @ndelangen what's the best practice about `@types` in SB codebase? I added them in `dependencies` because some of the types I exported in this addon are based on `jest` (and other libs) ones. So if a SB user want to use SB type it should install `@types/jest` too... 🤷‍♂